CMakeLists.txt

```diff
@@ -2,6 +2,8 @@ cmake_minimum_required(VERSION 3.21)
 
 project(GuitarTrainer)
 
+include_directories(/usr/include/mkl/)
```

Comment:
This will break in non debian distribution

Reply:
@Michanowski9 your part